lalrpop/Cargo.toml

```diff
@@ -27,7 +27,7 @@ is-terminal = "0.4.2"
 itertools = { version = "0.10", default_features = false, features = ["use_std"] }
 petgraph = { version = "0.6", default_features = false }
 regex = { workspace = true, features = ["std"] }
-regex-syntax = { workspace = true, features = ["unicode"] }
```

Comment:
isn't this required to allow regex 1.8 + regex-syntax 0.6?

Reply:
Only if it's not inherited from the workspace. So I enabled it there for consistency.